R/init.R

```diff
@@ -231,6 +231,8 @@ init <- function(data,
       if (!is.null(landing_module)) {
         do.call(landing_module$server, c(list(id = "landing_module_shiny_id"), landing_module$server_args))
       }
+      session$userData$lockfile_process <- process
+      reactivePoll(1000, session, checkFunc = lockfile_status, valueFunc = function(){})
```

Comment:
This does not yet produce `logger::log_trace` messages.
TODO for the further investigation.

Potential helpful resources on the `callr`

https://www.youtube.com/watch?v=DTMVzK7iZFU&t=1629s
https://towardsdev.com/a-simple-workflow-for-async-shiny-with-callr-f32393b087be

Reply:
Had a chance to watch the youtube tutorial on callr today but didn't learn more than what we already knew

Reply:
Hey @gogonzo the `reactivePoll` wasnt the right tool (or I wasn't able to properly use it!).
I managed to execute `lockfile_creation` status with `reactiveTimer` + `observe()` + `$destroy()` combination in here 3c480a7be0524dbcaa30f2a2d6165069df813e9c